### PR TITLE
update npm links

### PR DESCRIPTION
**Pull request by dhruv-anand-aintech**
---
labels: 'vdb comparison'
---

## Checklist before requesting a review
- [x] I have followed the [contribution guidelines](https://github.com/superlinked/VectorHub/tree/main/docs/tools/vdb_table)
- [x] I have provided a reference for each attribute for which I'm adding a "support" claim.

### CODE REVIEW

File: docs/tools/vdb_table/data/myscale.json [hunk 1]
```diff
@@ -168,7 +168,7 @@
   },
   "npm_downloads": {
     "value": 0,
-    "source_url": "",
+    "source_url": "https://www.npmjs.com/package/@clickhouse/client",
```
Comment (svonava) [inline docs/tools/vdb_table/data/myscale.json:171]: My scale file, not clickhouse.

File: docs/tools/vdb_table/data/turbopuffer.json [hunk 2]
```diff
@@ -168,7 +168,7 @@
   },
   "npm_downloads": {
     "value": 0,
-    "source_url": "",
+    "source_url": "https://www.npmjs.com/package/turbopuffer-sdk",
```
Comment (svonava) [inline docs/tools/vdb_table/data/turbopuffer.json:171]: Is the @ needed like with the other ones?

File: docs/tools/vdb_table/data/txtai.json [hunk 3]
```diff
@@ -168,7 +168,7 @@
   },
   "npm_downloads": {
     "value": 0,
-    "source_url": "",
+    "source_url": "https://www.npmjs.com/package/txtai",
```
Comment (svonava) [inline docs/tools/vdb_table/data/txtai.json:171]: @ ?

**Review by svonava — COMMENTED** [review]
For the other DBs did we add client libs or only the case when the npm actually has the whole database and distributes it through npm running in JS?

File: docs/tools/vdb_table/data/turbopuffer.json (hunk 2 above)
Comment (AruneshSingh) [reply docs/tools/vdb_table/data/turbopuffer.json:171]: Both styles work - ex [aperturedb](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/data/aperturedb.json) has without @ and [azureai](https://github.com/superlinked/VectorHub/blob/main/docs/tools/vdb_table/data/azureai.json) has with

File: docs/tools/vdb_table/data/myscale.json (hunk 1 above)
Comment (svonava) [reply docs/tools/vdb_table/data/myscale.json:171]: Interesting - well, I think the purpose of the stats is to track the popularity of a specific solution. And even though myscale uses the actual clickhouse client, attributing the clickhouse client popularity to myscale doesn't seem right.

**Comment by svonava** [comment]
> For the other DBs did we add client libs or only the case when the npm actually has the whole database and distributes it through npm running in JS?

@dhruv-anand-aintech what do you think about this aspect of client vs core? we should at the minimum stay consistent across the vendors on this decision.

**Comment by dhruv-anand-aintech** [comment]
I would show the client library if that's the only thing present, since that's what indicates usage.

**Comment by svonava** [comment]
> I would show the client library if that's the only thing present, since that's what indicates usage.

Sounds good - so basically whatever the company has on npm will be picked up.

But still I'd keep it to just that company, instead of going across vendors (except the case where we have a separate row for the opensource core and the main company that offers it as managed cloud - e.g. for zilliz & milvus, it's ok to share the same client npm because both projects are managed by a single entity, for myscale & clickhouse it's different entities so I'd not conflate).

**Pull request merged**